lib/zclient.h

```diff
@@ -786,8 +798,13 @@ extern void zclient_redistribute(int command, struct zclient *, afi_t, int type,
 extern void zclient_redistribute_default(int command, struct zclient *,
 					 afi_t, vrf_id_t vrf_id);
 
-/* Send the message in zclient->obuf to the zebra daemon (or enqueue it).
-   Returns 0 for success or -1 on an I/O error. */
+/*
+ * Send the message in zclient->obuf to the zebra daemon (or enqueue it).
+ * Returns:
+ * -1 on a I/O error
+ *  0 data was successfully sent
+ *  1 data was buffered for future usage
```

Comment:
I don't super-love the raw values - could we have constants or an enum even, so it's more readable?

Reply:
fixed this